jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/python/PythonSamProjectWizard.kt

```diff
@@ -4,36 +4,27 @@
 package software.aws.toolkits.jetbrains.services.lambda.python
 
 import com.intellij.openapi.progress.ProgressIndicator
-import com.intellij.openapi.project.Project
 import com.intellij.openapi.roots.ModifiableRootModel
+import com.intellij.openapi.ui.TextFieldWithBrowseButton
 import com.intellij.openapi.vfs.VirtualFile
 import com.intellij.util.PlatformUtils
 import software.amazon.awssdk.services.lambda.model.Runtime
 import software.aws.toolkits.jetbrains.services.lambda.BuiltInRuntimeGroups
 import software.aws.toolkits.jetbrains.services.lambda.sam.SamCommon
-import software.aws.toolkits.jetbrains.services.lambda.sam.SamSchemaDownloadPostCreationAction
 import software.aws.toolkits.jetbrains.services.lambda.wizard.IntelliJSdkSelectionPanel
 import software.aws.toolkits.jetbrains.services.lambda.wizard.SamNewProjectSettings
-import software.aws.toolkits.jetbrains.services.lambda.wizard.SamProjectGenerator
 import software.aws.toolkits.jetbrains.services.lambda.wizard.SamProjectTemplate
 import software.aws.toolkits.jetbrains.services.lambda.wizard.SamProjectWizard
-import software.aws.toolkits.jetbrains.services.lambda.wizard.SchemaResourceSelectorSelectionPanel
-import software.aws.toolkits.jetbrains.services.lambda.wizard.SchemaSelectionPanel
-import software.aws.toolkits.jetbrains.services.lambda.wizard.SdkSelectionPanel
+import software.aws.toolkits.jetbrains.services.lambda.wizard.SdkSelector
 import software.aws.toolkits.jetbrains.services.lambda.wizard.TemplateParameters
 import software.aws.toolkits.jetbrains.services.lambda.wizard.TemplateParameters.AppBasedTemplate
 import software.aws.toolkits.jetbrains.services.lambda.wizard.TemplateParameters.LocationBasedTemplate
-import software.aws.toolkits.jetbrains.services.schemas.SchemaCodeLangs
 import software.aws.toolkits.resources.message
-import java.nio.file.Paths
 
 class PythonSamProjectWizard : SamProjectWizard {
-    override fun createSchemaSelectionPanel(generator: SamProjectGenerator): SchemaSelectionPanel =
-        SchemaResourceSelectorSelectionPanel(generator.builder, generator.defaultSourceCreatingProject)
-
-    override fun createSdkSelectionPanel(generator: SamProjectGenerator): SdkSelectionPanel = when {
-        PlatformUtils.isPyCharm() -> PyCharmSdkSelectionPanel(generator.step)
-        else -> IntelliJSdkSelectionPanel(generator.builder, BuiltInRuntimeGroups.Python)
+    override fun createSdkSelectionPanel(projectLocation: TextFieldWithBrowseButton?): SdkSelector? = when {
+        PlatformUtils.isPyCharm() -> PyCharmSdkSelectionPanel(projectLocation)
+        else -> IntelliJSdkSelectionPanel(BuiltInRuntimeGroups.Python)
```

Comment:
Looking at this again, maybe we want the "pycharm/light" case to be the default?

Reply:
Probably safer that way